adapters/adapter.go

```diff
@@ -83,7 +83,7 @@ func For(task models.TaskSpec, store *store.Store) (AdapterWithMinConfs, error)
 	}
 	wa := MinConfsWrappedAdapter{
 		Adapter:                 ac,
-		ConfiguredConfirmations: store.Config.TaskMinConfirmations,
+		ConfiguredConfirmations: store.Config.MinOutgoingConfirmations,
```

Comment:
Perhaps my understanding is incorrect, but aren't many of these swapped?

We are waiting on enough confirmations to have the confidence to run tasks with side effects:
`TaskMinConfirmations` -> `MinIncomingConfirmations`

We are waiting on enough confirmations to believe that our sent (outgoing) eth transaction as been written to the blockchain:
`TxMinConfirmations` -> `MinOutgoingConfirmations`